pkgs/by-name/op/openmpi/package.nix

```diff
@@ -85,11 +73,15 @@ stdenv.mkDerivation (finalAttrs: {
     SOURCE_DATE_EPOCH = "0";
   };
 
-  outputs = [
-    "out"
-    "man"
-    "dev"
-  ];
+  # Required for build with gcc-14
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=int-conversion";
+
+  outputs =
+    [ "out" ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [
+      "man"
+      "dev"
+    ];
```

Comment:
This shouldn't be edited - probably a merge conflict resolution mistake. See https://github.com/NixOS/nixpkgs/pull/389036 .

Reply:
Yep.